jmh-benchmarks/src/main/java/org/apache/kafka/jmh/streams/MockMemoryLRUCache.java

```diff
@@ -0,0 +1,275 @@
+/**
```

Comment:
@ewencp - I did this so I could inject the org.github.jamm.MemoryMeter object for benchmarking  tracking cache size by memory.